fdb/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:latest
+USER root
+ARG SERVICE
+LABEL org.freva.service="$SERVICE"
+LABEL org.opencontainers.image.authors="DRKZ-CLINT"
+LABEL org.opencontainers.image.source="https://github.com/freva-org/freva-service-config"
+ENV    PYTHONUNBUFFERED=1 \
+       IN_DOCKER=1 \
+       SERVICE=${SERVICE} \
+       DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && apt-get install -y \
+    fdb5 \
+    libfdb5-dev 
+
+WORKDIR /fdb-test
```

Comment:
How about sticking to the structure in the other containers? That would be `/data/db` for data, `/data/config` for configuration.